web_server/login.py

```diff
@@ -0,0 +1,26 @@
+from flask import Blueprint, render_template, request, redirect, url_for
+
+login_page = Blueprint('login', __name__, template_folder='templates')
+
+users = {'sdv': {'email': 'sdv', 'phone number': 'sdv', 'password': 'sdv'}}
```

Comment:
Do we want to push the test-data? 
Maybe just leave it an empty dictionary (and add a comment about its structure if you'd like).